src/helpers/http/network.rs

```diff
@@ -137,33 +137,25 @@ mod tests {
     fn localhost_peers(h1_port: u16, h2_port: u16, h3_port: u16) -> Conf {
         let peer_discovery_str = format!(
             r#"
-[h1]
-    [h1.http]
+[http]
+    [http.h1]
```

Comment:
I actually liked the way config was written before. When I config things, I prefer to have one section for every node/role I am configuring and subsections for the specific functionalities. But code needs the transposed view of it - I wonder if it is possible to keep the previous format without too much extra work?

Reply:
sure. let me try reverting the commit